src/block.hpp

```diff
@@ -95,14 +95,15 @@ class BlockGrid
         ar(m_blockArchetypes);
     }
 
+    [[nodiscard]] std::uint32_t calculateIndex(const sf::Vector2u& pos) const;
```

Comment:
why public, delete getBlockIdx tbh lmao

Reply:
I needed it somewhere else, but I'm making a generic grid class anyways